LocalSettings.php

```diff
@@ -2860,6 +2860,23 @@
 				'edittemplateprotected' => true,
 			],
 		],
+		'+nycsubwaywiki' => [
+			'steward' => [
+				'managewiki-restricted' => true,
```

Comment:
Why assign `managewiki-restricted` when the global group already has the right? Any local ManageWiki changes would be logged locally either way.